tsconfig.json

```diff
@@ -1,20 +1,25 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "outDir": "lib",
+    "target": "es2015",
     "lib": ["dom", "dom.iterable", "esnext"],
     "allowJs": true,
     "skipLibCheck": true,
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
-    "strict": true,
+    "strict": false,
```

Comment:
Ajj, that's a pity. Is there a good reason to move away from strict? If this is a temporary necessity in order to prevent the scope of this PR to blow out of proportion, this might be something to put in a Jira ticket (including some starter info) , or perhaps a GitHub issue? It'd be nice to have a groomed/refined backlog available:)

Reply:
Although the strict flag was enabled the code was not. Think it is a good first step to enable the compiler to have some checks enabled. Next step can be enable the strict mode again.